flink-runtime/src/test/java/org/apache/flink/runtime/blob/BlobCacheGetTest.java

```diff
@@ -237,10 +237,14 @@ private void testGetFailsIncoming(@Nullable final JobID jobId, BlobKey.BlobType
             // make sure the blob cache cannot create any files in its storage dir
             if (blobType == PERMANENT_BLOB) {
                 tempFileDir =
-                        cache.getPermanentBlobService().createTemporaryFilename().getParentFile();
+                        ((PermanentBlobCache) cache.getPermanentBlobService())
```

Comment:
Are these casts actually necessary? Given that the test works against the implementation and not the interface I would've assumed that it should still work.

Reply:
Yes, this is not correct. I'll remove it.